meshroom/nodes/aliceVision/LDRToHDR.py

```diff
@@ -7,76 +7,88 @@ class LDRToHDR(desc.CommandLineNode):
     commandLine = 'aliceVision_convertLDRToHDR {allParams}'
 
     inputs = [
-        desc.File(
-            name='input',
-            label='Input',
-            description="List of LDR images or a folder containing them ",
-            value='',
+        desc.ListAttribute(
+            elementDesc=desc.File(
+                name='inputFolder',
+                label='Input File/Folder',
+                description="Folder containing LDR images",
+                value='',
+                uid=[0],
+                ),
+            name="input",
+            label="Input Files or Folders",
+            description='Folders containing LDR images.',
+        ),
+        desc.BoolParam(
+            name='fisheyeLens',
+            label='Fisheye Lens',
+            description="Check if fisheye lens",
```

Comment:
"Enable if a fisheye lens has been used. This will improve the estimation of the Camera's Response Function by considering only the pixels in the center of the image and thus ignore undefined/noisy pixels outside the circle defined by the fisheye lens."